### DISCUSSION TITLE
What happens if I move a file from the library?

### DISCUSSION THREAD

**Discussion by miguelarios** [Tzahi12345/YoutubeDL-Material#637] [Q&A] [answered]
I want to use this app to mostly download and then some videos I will reorganize into plex or another app (embed in Obsidian as a helpful resource for some documentation etc.). In the past, when I tried this, there appeared an artifact of the file. I can try to move it myself but since I have yet to figure out how or where to move my files in plex to organize with metadata I am trying to ask proactively.

**Comment by GlassedSilver** [comment, marked as answer]
You end up with a database that points to entries that no longer exist.

Those entries can be safely deleted. This PR brought a maintenance dashboard that includes a task to look for missing files and clear their reference from the DB: #554

**Comment by maanich** [comment]
how can i enter downloader folder i instlled and using gui download a file but cant able to access where  this video downloading.
059157e76378   tzahi12345/youtubedl-material:latest   "/app/entrypoint.sh …"   21 minutes ago   Up 21 minutes                              
what command to get access to the video file to transfer to other place

**Reply by Tzahi12345** [reply]
You should have mounted the `users` (if multi-user mode) or `video` folder (if not). The file will be in one of those two folders
